docs/Consuming.md

```diff
@@ -6,25 +6,61 @@ To contribute/develop, see [here](./Contributing.md).
 
 ## Running the Platform Locally
 
-1. Initialize KAS Keys ```.github/scripts/init-temp-keys.sh -o kas-keys```
-1. Stand up the local Postgres database and Keycloak instances using `docker compose up -d --wait`.
-1. Copy the `opentdf-example.yaml` file to `opentdf.yaml` and update the [configuration](./docs/Configuring.md) as needed.
-1. Bootstrap Keycloak
+> [!WARNING]
+> This quickstart guide is intended for development and testing purposes only. The OpenTDF platform team does not
+> provide recommendations for production deployments.
 
-   ```sh
-   docker run --network opentdf_platform \
-         -v "$(pwd)/opentdf.yaml:/home/nonroot/.opentdf/opentdf.yaml" \
-         -it registry.opentdf.io/platform:nightly provision keycloak -e http://keycloak:8888/auth
+To get started with the OpenTDF platform make sure you are running the same Go version found in the `go.mod` file.
+
+<!-- markdownlint-disable MD034 github embedded sourcecode -->
+https://github.com/opentdf/platform/blob/main/service/go.mod#L3
+
+> **Note for Apple M4 chip users:**  
+> If you are running on an Apple M4 chip, set the Java environment variable before running any commands:
+> ```sh
+> export JAVA_OPTS_APPEND="-XX:UseSVE=0"
+> ```
+> This resolves SIGILL with Code 134 errors when running Java processes.
+
+
+1. **Initialize Platform Configuration**
+   ```shell
+   cp opentdf-dev.yaml opentdf.yaml
+   perl -i -pe 's/e1/ec1/g' opentdf.yaml
+   # Requires yq to be installed (see https://github.com/mikefarah/yq#install)
+   yq eval '.services.kas.ec_tdf_enabled = true' -i opentdf.yaml
```

Comment:
Do we want to enable this? Maybe discuss how experimental features are enabled generally here

Reply:
I am not sure. I referred here: https://github.com/opentdf/tests/blob/main/xtest/README.md#platform-backend, which helped me get the service started locally.